evaluators/langevals/langevals_langevals/grapheval.py

```
@@ -0,0 +1,325 @@
+):
+    """
+    Allows you to check for hallucinations by utilizing Knowledge Graphs
+    """
+
+    name = "GraphEval"
+    category = "custom"
+    default_settings = GraphEvalSettings()
+    is_guardrail = True
+
+    def evaluate(self, entry: GraphEvalEntry) -> SingleEvaluationResult:
+        details = None
```

Ruff (0.8.2)

Local variable `details` is assigned to but never used

Remove assignment to unused variable `details`

(F841)